ArduPlane/quadplane.cpp

```diff
@@ -238,6 +238,14 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("ASSIST_ANGLE", 45, QuadPlane, assist.angle, 30),
 
+    // @Param: ASSIST_OPTIONS
+    // @DisplayName: Quadplane assistance options
+    // @Description: Options for special QAssist features
+    // @Bitmask: 0: Disable force fixed wing controller recovery
+    // @Bitmask: 1: Disable quadplane spin recovery
+    // @User: Standard
+    AP_GROUPINFO("ASSIST_OPTIONS", 47, QuadPlane, assist.options, 0),
```

Comment:
47 was tilt type per the comment on 249. We are still doing the conversion so we probably cant re-use it yet.

Reply:
we can re-use as it is a different type. In AP_Param::scan() we do a type check so reusing is fine as long as the type is different. The type was AP_int8 for tilt type, this is AP_Int16

Reply:
Ah, OK, it would be worth updating the comment on 249.

Reply:
added